proposals/0029-cooperative-vector.md

```diff
@@ -102,383 +111,336 @@ Introduce new HLSL intrinsics to accelarate matrix-vector operations. In this sp
 
 ## Detailed design
 
-### Intrinsics for Vector-Matrix Operations
+### Matrix-Vector Multiply and Multiply-Add Operations
```

Comment:
since this section also covers the OuterProductAccumulate and ReduceSumAccumulate, just call it "Vector-Matrix Operations"?

Reply:
nm. I see how you have structured the spec now.